src/components/image-card-grid.vue

```diff
@@ -0,0 +1,78 @@
+<template>
```

Comment:
We can but think it's out of scope for this PR since it means we need to change how we handle all graphql values right? Atm. there's no way to import fragments from external files.

Reply:
Alright, I'll create a separate ticket for that